arkane/statmech.py

```diff
@@ -316,7 +316,9 @@ def load(self, pdep=False, plot=False):
             except KeyError:
                 raise InputError('Model chemistry {0!r} not found in from dictionary of energy values in species file '
                                  '{1!r}.'.format(self.modelChemistry, path))
-        if not os.path.isfile(energy.path):
+        if not hasattr(energy, 'path'):
+            pass	
+        elif not os.path.isfile(energy.path):
```

Comment:
I would suggest putting the check in the same statement: `if hasattr(energy, 'path') and not os.path.isfile(energy.path)`

Also, I'm not sure if Alon has opinions on use of `hasattr` vs checking `isinstance(energy, Log)`.

Reply:
I think `isinstance(energy, Log)` is cleaner, and it should definitely be in the same statement.
@jeehyunatrmg, we can do it together tomorrow if you'd like, or simply go-ahead and make these fixes.